test/gui/shared/steps/steps.py

```diff
@@ -102,14 +102,24 @@ def step(context, displayname, host):
     )
 
 
+def getUserInfo(context, username, attribute):
+    # add and update users to the global createdUsers dict if not already there
+    # so that we don't have to request for user information in every scenario
+    # but instead get user information from the global dict
+    global createdUsers
+    if createdUsers and username in createdUsers:
```

Comment:
why if createdUsers...? Can't we do 
```suggestion
    if username in createdUsers:
```

Reply:
Previously, `createdUsers` was intialized with `None` but now it is initialized with `{}` so we can do it.